### DISCUSSION TITLE
showing/import PLY file in c#

### DISCUSSION THREAD

**Discussion by Xander-Vreeswijk** [dotnet/csharplang#3890] [General]
I have made a point cloud and saved it in an PLY file but I don't know how to show the ply file in c#. Can someone help me with this problem?

**Comment by orthoxerox** [comment]
Please use [other communities](https://github.com/dotnet/csharplang/blob/master/Communities.md) to ask questions about the *use* of C#/.Net/3rd party libraries. These discussions are for questions/ideas about the *design* of the C# language itself and its evolution.
